flink/v1.19/flink/src/main/java/org/apache/iceberg/flink/sink/shuffle/SketchRangePartitioner.java

```diff
@@ -47,18 +46,6 @@ class SketchRangePartitioner implements Partitioner<RowData> {
   public int partition(RowData row, int numPartitions) {
     // reuse the sortKey and rowDataWrapper
     sortKey.wrap(rowDataWrapper.wrap(row));
-    int partition = Arrays.binarySearch(rangeBounds, sortKey, comparator);
```

Comment:
this refactoring is to make it easier to write unit test with the static util method